sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -321,6 +321,19 @@ static int mt8195_dsp_suspend(struct snd_sof_dev *sdev, u32 target_state)
 {
 	struct platform_device *pdev = container_of(sdev->dev, struct platform_device, dev);
 	int ret;
+	u32 reset_sw, dbg_pc;
```

Comment:
Your commit message has long sentences, repeats and missing spaces. It's hard to read!

suggested edit - please double-check if the wording is technically correct.

````
During suspend flow, sof_suspend will be called and the pm_ops->ctx_save callback notifies DSP of the upcoming power down.

Upon receipt of the ctx_save IPC, the DSP will start the D3 transition. When the DSP is idle, an interrupt is generated to notify the host of the power state change.

Since the host and DSP are two different processors, there could be a race condition, which can be avoided by polling with 1s timeout and 500us intervals 
````

Reply:
@plbossart  Thanks a lot for suggestion. I modify some description in the bold part. Interrupt should be sent before DSP enter idle. Thanks.
Upon receipt of the ctx_save IPC, the DSP will start the D3 transition.
 **Before the DSP enter idle**, an interrupt is generated to notify the host of the power state change.